Tasks/VsTestPlatformToolInstaller/task.loc.json

```diff
@@ -14,7 +14,11 @@
   "version": {
     "Major": 1,
     "Minor": 0,
+<<<<<<< HEAD
+    "Patch": 2
+=======
```

Comment:
same..

Reply:
must've happened after the latest merge, but I remember resolving and committing on vscode